common/lib/xmodule/xmodule/public/js/vertical_student_view.js

```diff
@@ -0,0 +1,16 @@
+/* JavaScript for Vertical Student View. */
+window.VerticalStudentView = function (runtime, element) {
+
+    RequireJS.require(['js/bookmarks/views/bookmark_button'], function (BookmarkButton) {
```

Comment:
@andy-armstrong That numbering scheme had something to do with: https://openedx.atlassian.net/browse/PLAT-110. I don't think that effects this RequireJS usage.